awx/main/models/credential/__init__.py

```diff
@@ -559,7 +559,12 @@ def build_extra_vars_file(vars, private_dir):
 
             if extra_vars:
                 path = build_extra_vars_file(extra_vars, private_data_dir)
-                args.extend(['-e', '@%s' % path])
+                # FIXME: develop some better means of referencing paths inside containers
+                container_path = os.path.join(
+                    '/runner',
+                    os.path.basename(path)
+                )
+                args.extend(['-e', '@%s' % container_path])
```

Comment:
@ryanpetrello just to let you know what's going on for running jobs in containers. We can't ever reference any absolute paths 😝 

This won't be the last of this kind of finding... but clearly since this broke all credentials that used extra vars its impact was relatively high. Just off the top of my head, I would guess that credentials that use an env var to reference a file would also not work.